src/sinopiaServer.js

```diff
@@ -65,24 +66,37 @@ export const getResourceTemplate = (templateId) => {
     return emptyTemplate
   }
 
+  // TODO: Replace the current return value with a promise, so both functions
```

Comment:
I would like to write this up as a new issue and ask either @jgreben or @jermnelson to take a crack at it.

Reply:
👍